src/main/java/it/pagopa/pn/paperchannel/service/impl/PaperTenderServiceImpl.java

```diff
@@ -64,6 +63,22 @@ public Mono<PnPaperChannelCostDTO> getSimplifiedCost(String cap, String zone, St
                 );
     }
 
+    @Override
+    public Mono<PnPaperChannelCostDTO> getCostFromTenderId(String tenderId, String geokey, String productType) {

```

Comment:
Non riusciamo a centralizare la parte comune con il metodo getSimplifiedCost? COsì evitiamo duplciazione. Alla fine cambia solo che in input abbiamo il tednerId...

Reply:
La centralizzazione è stata effettuata.